packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -40,6 +42,8 @@ export default Component.extend({
    */
   classNames: [ 'table-widget' ],
 
+  bardMetadata: service(),
```

Comment:
Can we add jsdoc here pls?

Reply:
it seems that other place also do not have jsdoc for this